securedrop_client/logic.py

```diff
@@ -374,9 +379,13 @@ def sync_api(self):
             logger.debug("You are authenticated, going to make your call")
 
             job = MetadataSyncJob(self.data_dir, self.gpg)
-
             job.success_signal.connect(self.on_sync_success, type=Qt.QueuedConnection)
-            job.failure_signal.connect(self.on_sync_failure, type=Qt.QueuedConnection)
+            # Handle failure depending on whether or not the sync originated from a manual refresh
+            if manual_refresh:
+                job.failure_signal.connect(self.on_refresh_failure, type=Qt.QueuedConnection)
+            else:
+                job.failure_signal.connect(self.on_sync_failure, type=Qt.QueuedConnection)
```

Comment:
Here we can (tbh I think this is optional now but I'll let you decide if you want to add it) increase the number of tries for the sync job _not_ triggered by the user since in that case it doesn't really matter if they get rapid feedback. We can do this by setting `remaining_attempts` explicitly on the job.